api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -105,6 +113,15 @@
         this.participantCounter = participantCounter;
     }
 
+    @Override
+    public ResponseEntity<CohortAnnotationDefinition> createCohortAnnotationDefinition(String workspaceNamespace,
+                                                                                       String workspaceId,
+                                                                                       Long cohortId,
+                                                                                       Long cdrVersionId,
```

Comment:
I don't think we want CDR version ID here... cohort annotation definitions can be scoped to a cohort, but probably should stick around if you start reviewing a new CDR version. That's why CDR version / cohort review isn't a part of the key for the definition table above.

(Ideally we'd have a way of sharing them across cohorts or even workspaces, but that's another matter...)

Reply:
Agreed, this seems logical to me. See my comment below about splitting the api calls to make sure I'm understanding you correctly.